### PR TITLE
Update message

### PR DESCRIPTION
**Pull request by Fanevanjanahary**

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 23 min

File: django_project/frontend/src/containers/MainPage/Upload/SpeciesUploader.tsx [hunk 1]
```diff
@@ -188,9 +188,10 @@ export default function Uploader(props: UploaderInterface) {
             })
             }).then( response => {
                 if (response.ok) {
-                    wait(500).then(r =>
+                    wait(5000).then(r =>
                     axios.get(`${STATUS_URL}${session}/`).then((response)=>{
                     if (response.data) {
+                        console.log(response.data)
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Upload/SpeciesUploader.tsx:194]: Please remove this

File: django_project/frontend/src/containers/MainPage/Upload/SpeciesUploader.tsx [hunk 2]
```diff
@@ -188,9 +188,10 @@ export default function Uploader(props: UploaderInterface) {
             })
             }).then( response => {
                 if (response.ok) {
-                    wait(500).then(r =>
+                    wait(5000).then(r =>
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/Upload/SpeciesUploader.tsx:191]: Why do we need to wait for 5 seconds?

**Review by dimasciput — CHANGES_REQUESTED** [review]

**Review by dimasciput — APPROVED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 19 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 1 min 16 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 38 sec

**Pull request merged**